src/Symfony/Bridge/Monolog/Handler/ConsoleHandler.php

```diff
@@ -40,9 +83,15 @@
  * This mapping can be customized with the $verbosityLevelMap constructor parameter.
  *
  * @author Tobias Schultze <http://tobion.de>
+ *
+ * @final
```

Comment:
```suggestion
 * @final since Symfony 6.1
```

If we softly flag a class as `@final` or `@internal` in a minor version, we usually add the Symfony version that introduced that change. This reminds us that we cannot treat the method as final/internal until the next major release.